reporting/tests/src/test/java/org/jboss/windup/reporting/CSVExportingTest.java

```diff
@@ -0,0 +1,210 @@
+package org.jboss.windup.reporting;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang.RandomStringUtils;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.forge.arquillian.AddonDependencies;
+import org.jboss.forge.arquillian.AddonDependency;
+import org.jboss.forge.arquillian.archive.AddonArchive;
+import org.jboss.forge.furnace.util.Predicate;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.windup.config.RuleProvider;
+import org.jboss.windup.config.phase.MigrationRulesPhase;
+import org.jboss.windup.config.phase.ReportGenerationPhase;
+import org.jboss.windup.config.phase.ReportRenderingPhase;
+import org.jboss.windup.exec.WindupProcessor;
+import org.jboss.windup.exec.configuration.WindupConfiguration;
+import org.jboss.windup.exec.rulefilters.NotPredicate;
+import org.jboss.windup.exec.rulefilters.RuleProviderPhasePredicate;
+import org.jboss.windup.graph.GraphContext;
+import org.jboss.windup.graph.GraphContextFactory;
+import org.jboss.windup.graph.model.ProjectModel;
+import org.jboss.windup.graph.model.resource.FileModel;
+import org.jboss.windup.graph.service.LinkService;
+import org.jboss.windup.graph.service.ProjectService;
+import org.jboss.windup.reporting.model.ApplicationReportIndexModel;
+import org.jboss.windup.reporting.model.ClassificationModel;
+import org.jboss.windup.reporting.model.InlineHintModel;
+import org.jboss.windup.reporting.service.ApplicationReportService;
+import org.jboss.windup.reporting.service.ClassificationService;
+import org.jboss.windup.reporting.service.InlineHintService;
+import org.jboss.windup.rules.apps.java.config.ScanPackagesOption;
+import org.jboss.windup.rules.apps.java.config.SourceModeOption;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import javax.inject.Inject;
+import java.io.*;
```

Comment:
@mbriskar Just FYI, I think the intellij default is probably causing this. Go to Settings, "Editor:Code Style:Java" then the "Imports" tabs. The default for "Class count to use import with '*'" is probably too low.